pkg/resmgr/flags.go

```diff
@@ -72,7 +73,7 @@ func init() {
 		"Minimum interval between two container rebalancing attempts. Use 'disable' for disabling.")
 	flag.StringVar(&opt.StateDir, "state-dir", "/var/lib/nri-resource-policy",
 		"Permanent storage directory path for the resource manager to store its state in.")
-
+	flag.BoolVar(&opt.E2ETestHTTPQuery, "enable-e2e-test-http-query", false, "Allow query via HTTP when doing e2e testing.")
```

Comment:
nitpick: I think this command line option name could be more descriptive. How about something like:
```
    flag.BoolVar(&opt.enableTestAPIs, "enable-test-apis", false, "Allow enabling various test APIs (currently only 'e2e-test' test controller).")
```